sw/vendor/patches/riscv_compliance/0002-Add-OpenTitan-target.patch

```diff
@@ -298,7 +298,7 @@ index 0000000..a44e0c3
 +        ( test -e "$(OT_FPGA_UART)" || ( echo "UART device '$(OT_FPGA_UART)' not available. Set OT_FPGA_UART." >&2 && exit 1 ) ) \
 +        && echo "Writing '$(<).bin' to device." \
 +        && $(OT_BIN)/sw/host/spiflash/spiflash --input="$(<).bin" \
-+        && stty -F "$(OT_FPGA_UART)" speed 230400 cs8 -cstopb -parenb \
++        && stty -F "$(OT_FPGA_UART)" speed 115200 cs8 -cstopb -parenb \
```

Comment:
Can you make those changes in the patch repo (https://github.com/lowRISC/riscv-compliance/commits/ot) as well (or let me know and I'll do it). (Rebase the second patch to include the changes.)

I'll try to remember next time I work on riscv-compliance to make the baud rate a variable, but no need to do that now.

Reply:
Alright, that's now done as well. I did a `--refresh-patches` on the vendored in RISC-V compliance suite. Both the sources and the patches are updated (also new hashes etc).